frontend/src/views/IssueDetail.vue

```diff
@@ -1171,9 +1175,54 @@ export default {
       return count > 1;
     });
 
+    const database = computed((): Database | undefined => {
+      const temp = selectedStage as unknown;
+      if (create) {
+        const stage = temp as ComputedRef<StageCreate>;
+        const databaseId = stage.value.taskList[0].databaseId;
+        if (databaseId) {
+          return store.getters["database/databaseById"](databaseId);
+        }
+
+        return undefined;
+      }
+      const stage = temp as ComputedRef<Stage>;
+      return stage.value.taskList[0].database;
+    });
+
+    const instance = computed((): Instance => {
+      const temp = selectedStage as unknown;
+      if (create) {
+        // If database is available, then we derive the instance from database because we always fetch database's instance.
+        // On the other hand, instance for stage.taskList[0].instanceId might not be loaded (e.g. when creating an update schema issue)
+        if (database.value) {
+          return database.value.instance;
+        }
+        const stage = temp as ComputedRef<StageCreate>;
+        return store.getters["instance/instanceById"](
+          stage.value.taskList[0].instanceId
+        );
+      }
+      const stage = temp as ComputedRef<Stage>;
+      return stage.value.taskList[0].instance;
+    });
+
+    const sqlHint = computed(() => {
+      if (!create && selectedMigrateType.value == "BASELINE") {
+        return `This is a baseline migration and bytebase won't apply the SQL to the database, it will only record a baseline history`;
+      }
+      if (instance.value.engine === "SNOWFLAKE" && create) {
```

Comment:
I think this should apply to non-create mode as well (e.g. user may forget to specify the the schema upon creation, we need to hint her in the normal mode)